node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'node:fs/promises';
+
+try {
+  const number = Math.random();
+  const text = number.toString();
```

Comment:
This conversion isn't actually necessary since JS will convert it to a string when you do the string interpolation on the next line.